core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -16,7 +16,11 @@
 	String updateCartItemPath = "/us/V1/guest-carts/{cartId}/items/{ItemId}";
 	String emptyCartPah = "/us/V1/guest-carts";
 	String guestCartItemDeletePah = "/us/V1/guest-carts/{cartId}/items/{itemId}";
+
     String servicePathCartUpdateItems = "/V1/guest-carts/";
+    String guestCartShippingInformation = "/us/V1/guest-carts/{cartId}/shipping-information";
```

Comment:
@nitheshkumar-hcl  append Path in variable name.

Reply:
variable name is updated and committed the changes